doc/design/quic-programming-guide.md

```diff
@@ -126,11 +126,12 @@ Please follow [Conference Server build instructions](https://github.com/open-web
 Steps to run Conference Server with pre-built binary:
 
 1. Go to the untarred conference server dir, and run `./bin/init-all.sh --deps`; this would try to install mongodb and rabbitmq-server. Don't set any password for rabbitmq or mongodb.
-2. Still in the same dir, run `bin/init-all.sh`.
-3. Update quic_agent/agent.toml, set `hostname` to the hostname quic agent is running on.
-4. run `bin/start-all.sh`.
-5. Open [https://localhost:3004/quic.html](https://localhost:3004/quic.html) on the same host with Chrome browser to visit the web sample page. Due to the test certificate, you may need confirm this unsafe access.
-6. Press 'Start Sending' button to start transferring data to conference server. Press 'Stop Sending' button on the web page to stop sending.. If there's no error in the console, that implies server is correctly setup.
+1. Still in the same dir, run `bin/init-all.sh`.
+1. Update quic_agent/agent.toml, set `hostname` to the hostname quic agent is running on.
```

Comment:
"1." -> "2."?

Reply:
Markdown automatically increases the number. Numbering all of them as 1 so we don't need to modify the number when a new item is inserted.

Preview: https://github.com/open-webrtc-toolkit/owt-server/blob/352963d338cadf087687bb864d774d57a8bc96f2/doc/design/quic-programming-guide.md#how-to-use-pre-built-conference-server-binary.